components/builder-worker/src/runner/docker.rs

```diff
@@ -145,7 +145,12 @@ impl<'a> DockerExporter<'a> {
         );
         cmd.env(DOCKER_HOST_ENVVAR, sock); // Use the job-specific `dockerd`
         cmd.stdout(Stdio::piped());
-        cmd.stderr(Stdio::piped());
+        match env::var_os(DEV_MODE) {
+            Some(_) => debug!("Not exporting in dev mode"),
```

Comment:
Sneaky :)  Maybe put in a comment that this part is a workaround for now until we solve the streaming log problem correctly?